libs/api-types/src/geometries.ts

```diff
@@ -0,0 +1,16 @@
+import type { MultiPolygon, Polygon } from 'geojson'
+
+export interface ContextAreaGeometryProperties {
+  fid: string
+  mrgid: string
```

Comment:
Are `fid` and `mrgid` actually needed at some point? What do they represent?

Reply:
Nop, they are not used anywhere but as they are in the api response I wanted to type properly